Source/Diagnostics/WarpXIO.cpp

```diff
@@ -407,7 +407,7 @@ WarpX::InitFromCheckpoint ()
 std::unique_ptr<MultiFab>
 WarpX::GetCellCenteredData() {
 
-    BL_PROFILE("WarpX::GetCellCenteredData");
+    BL_PROFILE_REGION("GetCellCenteredData()");
```

Comment:
I added BL_PROFILE_REGION when I was timing GetCellCenteredData() to see how much time it takes and if using a slice to obtain the CellCenteredData() will help reduce the time. Because as of now -- the steps performed for the full 3D snapshot and slice is the same. It was only for my own analysis -- I will revert it back.